internal/services/appservice/helpers/web_app_schema.go

```diff
@@ -247,6 +251,14 @@ func SiteConfigSchemaWindows() *pluginsdk.Schema {
 					Optional: true,
 				},
 
+				"health_check_eviction_time": {
```

Comment:
should this be 
```suggestion
				"health_check_eviction_time_in_min": {
```
to be explicit?

Reply:
Makes sense 👍